.github/workflows/ci.yaml

```diff
@@ -25,11 +25,14 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: ['ubuntu', 'windows', 'macos']
+        os: ['ubuntu', 'macos']
```

Comment:
Is it absolutely necessary to drop Windows support?

Reply:
> Main issue is that I don't think its good to drop Windows support entirely.
Yikes didn't see that change on first reivew.

I agree with you here @dwhswenson. Plus I think that Windows is pretty important for industrial users too, but it seems like OFF has just decided not bother: https://open-forcefield-toolkit.readthedocs.io/en/latest/installation.html#os-support

Maybe in the first instance we drop support, and check if it's a deal breaker on Tuesday? Mainly saying this as "we probably want this in before Tuesday, and dealing with whatever breaks in CI before the release is going to be more time than we probably can afford it".

Reply:
Fair enough... I was hoping that this was maybe an OpenEye issue, not OpenFF.

As I recall, our plans specifically state that we will have support for using OpenFF. I don't remember anything about Windows support. 🤷‍♂️ Calling this resolved, with the hopes that someday Windows support comes back.